ur_simulation_gz/doc/index.rst

```diff
@@ -0,0 +1,224 @@
+        allow_partial_joints_goal: false
+        constraints:
+          stopped_velocity_tolerance: 0.2
+          goal_time: 0.0
+          shoulder_pan_joint: { trajectory: 0.2, goal: 0.1 }
+          shoulder_lift_joint: { trajectory: 0.2, goal: 0.1 }
+          elbow_joint: { trajectory: 0.2, goal: 0.1 }
+          wrist_1_joint: { trajectory: 0.2, goal: 0.1 }
+          wrist_2_joint: { trajectory: 0.2, goal: 0.1 }
+          wrist_3_joint: { trajectory: 0.2, goal: 0.1 }
+
+    forward_velocity_controller:
+      ros__parameters:
+        joints:
+          - alice_shoulder_pan_joint
+          - alice_shoulder_lift_joint
+          - alice_elbow_joint
+          - alice_wrist_1_joint
+          - alice_wrist_2_joint
+          - alice_wrist_3_joint
+        interface_name: velocity
+
+    forward_position_controller:
+      ros__parameters:
+        joints:
+          - alice_shoulder_pan_joint
+          - alice_shoulder_lift_joint
+          - alice_elbow_joint
+          - alice_wrist_1_joint
+          - alice_wrist_2_joint
+          - alice_wrist_3_joint
+
+To load the newly defined file, it is possible to specify its absolute path with the ``controllers_file`` argument. Together with it the desired prefix should be also be specified as argument, like in the following example:
+
+.. code-block:: bash
+
+   $ ros2 launch ur_simulation_gz ur_sim_control.launch.py ur_type:=ur10e tf_prefix:="alice_" controllers_file:="/home/ubuntu/ur_controllers_test.yaml"
+
+**Note**: the ``tf_prefix`` argument is not available for ``ur_sim_moveit.launch.py``, since it would require a custom definition of the moveit config package for properly setting it up.
+
+The last customization option allows to instantiate the robot in a proper setup instead of an empty world, like the given launch files do by default. The first step to create a complete simulation is to define a world file (.sdf): for this example we can create a simple custom world ``test_world.sdf`` and place it in ``ur_gz_simulation/worlds``. For more details about building worlds in Gazebo, it's possible to check the `related tutorial <https://gazebosim.org/docs/harmonic/sdf_worlds>`_.
+To use the new world changes are minimimal:
+
+- in ``ur_sim_control.launch.py`` we can add the path to the ``launch_setup`` as
+
+.. code-block:: python
+
+    world_file_path = PathJoinSubstitution(
+        [FindPackageShare("ur_simulation_gz"), "worlds", "test_world.sdf"]
+    )
+
+- and then modify the ``gz_launch_description`` in order to use the desired world file:
+
+.. code-block:: python
+
+   gz_launch_description = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [FindPackageShare("ros_gz_sim"), "/launch/gz_sim.launch.py"]
+        ),
+        launch_arguments={
+            "gz_args": IfElseSubstitution(
+                gazebo_gui, if_value=[" -r -v 4 ", world_file_path], else_value=[" -s -r -v 4 ", world_file_path]
+            )
+        }.items(),
+    )
```

Comment:
Removed these parts and added the file. 
Regarding the reason behind using a world file, it could be to add objects that you want to grasp/interact in your simulation. All the rest should be easily covered by using the URDF.